### PR TITLE
Mar 3468.4

### PR DESCRIPTION
**Pull request by Reshma-M-A**
Admin Teams support

In the team screen you will see :
1. Team Leads Table 
2. Team Member's Table 
3. Brief Ids associated with Teams

### CODE REVIEW

File: app/main/views/buyers.py [hunk 1]
```diff
@@ -268,11 +268,27 @@ def find_team_by_team_id():
...
+    team_leads = team.get('teamLeads')
+    team_leads_email_name = []
+
+    for i in team_leads:
+        team_leads_email_name.append(team_leads.get(i))
+
+    team_members = team.get('teamMembers')
+
+    team_members_email_name = []
+    team_members_permissions = []
```
Comment (ndavison) [inline app/main/views/buyers.py]: Doesn't look like this is being used.

Comment (Reshma-M-A) [reply app/main/views/buyers.py]: Good catch! - It is just team_members_permissions not being used

Removed now

File: app/templates/view_teams.html [hunk 2]
```diff
@@ -29,72 +29,63 @@ <h3>Team Details</h3>
...
+    field_headings=[
+      'Name',                    
+      'Email Address',
+    ],
+    field_headings_visible=True)
+%}
+  {% call summary.row() %}
+    {{ summary.text(item.name) }}
+    {% call summary.field() %}
+      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" method="get"  class="question">{{ item.emailAddress}}</a>
```
Comment (ndavison) [inline app/templates/view_teams.html]:
```suggestion
      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" class="question">{{ item.emailAddress}}</a>    
```

I don't think you need `method="get"` here.

File: app/templates/view_teams.html [hunk 3]
```diff
@@ -29,72 +29,63 @@ <h3>Team Details</h3>
...
+  'Create Work Orders'
+  ],
+  field_headings_visible=True)
+%}
+{% call summary.row() %}
 
-  <div>
+    {{ summary.text(item.name) }}
+    {% call summary.field() %}
+      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" method="get"  class="question">{{ item.emailAddress}}</a>
```
Comment (ndavison) [inline app/templates/view_teams.html]:
```suggestion
      <a href="{{ url_for('.find_user_by_email_address', email_address = item.emailAddress)}}" class="question">{{ item.emailAddress}}</a>    
```

File: app/templates/view_teams.html [hunk 4]
```diff
@@ -29,72 +29,63 @@ <h3>Team Details</h3>
...
-      {% endcall %}
-  </div>
+<h3> Team Members' Permissions</h3>
+{% call(item) summary.list_table(
+  team_members,
+  field_headings=[  
+  'Name',                   
+  'Email Address',
+  'Download Reports',
+  'Publish Opporunities',
```
Comment (ndavison) [inline app/templates/view_teams.html]:
```suggestion
  'Publish Opportunities',
```

File: app/templates/view_teams.html (hunk 2 above)
Comment (Reshma-M-A) [reply app/templates/view_teams.html]: removed get

File: app/templates/view_teams.html (hunk 3 above)
Comment (Reshma-M-A) [reply app/templates/view_teams.html]: removed get

**Pull request merged**